### DISCUSSION TITLE
Anlernen nicht möglich mit HmIP-RFUSB unter Proxmox

### DISCUSSION THREAD

**Discussion by silvermd** [OpenCCU/OpenCCU#2443] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

ich benötige leider Hilfe. Ich habe eine CCU2 im Einsatz. Hier habe eine Fensterkontakt HM-Sec-SCo abgelernt und wollte diesen im einem HomeAssistant-Addon RaspberryMatic anlernen, was nicht ging.

Ich dachte mir ok, zu viel Virtualisierung lass ich mal bleiben und habe es unter Proxmox direkt installiert, ohne eingespielte Ccu2 backups.
Hier einen Q-3 HmIP-RFUSB unter usb-0000:02:1b.0-1 durchgeschliefen.

In der Log sehe ich diesen auch. Der Stick ist an einem 2 meter USB Verlängerungskabel dran, ist das 3te Kabel was ich versucht habe. 

Ich hoffe ihr könnt folgende Log besser lesen und mir ein paar Ideen geben, ich komme gerade nicht weiter.

Randinfo: der HM-Sec-SCo lasst sich jederzeit normal an der Ccu2 an und ablernen (ist aus der Geräteliste raus)... zurückgesetzt habe ich ihn auch immer wieder.

Warum ich das ganze machen will, der Performace willen und weil ich mir eine HmIP-PSM-2 kaufen musste, die alte PSM1 hats leider entschärft, jedoch kann Ccu2 diese nicht anlernen, anscheint unbekannt.
Ich werde es am WE noch mit einem Rapberry 4 und wieder frischem Rasberrymatic probieren.

Ich bedanke mich für eure Zeit und hoffe es gibt eine Lösung, außer ein besseres Funkmodul zu kaufen.

```
***** /var/log/boot.log *****
fsck 1.46.5 (30-Dec-2021)
userfs: clean, 91/4112384 files, 337269/16449280 blocks
Starting watchdog...
Identifying host system: ova-KVM, OK
Initializing ZRAM Swap: OK
Initializing RTC Clock: onboard, OK
Starting acpid: OK
Running sysctl: OK
Running seedrng: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Populating /dev using udev: done
Init onboard LEDs: init, OK
Starting qemu-guest-agent: OK
Starting irqbalance: OK
Starting system message bus: done
Starting iptables: OK
Starting bluetoothd: disabled
Starting network: eth0: link up, static, firewall, inet up, 192.168.3.176, wlan0: missing, OK
Starting Network Interface Plugging Daemon: eth0.
Starting chrony: OK
Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:02:1b.0-1, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:02:1b.0-1, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.18, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting sshd: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: ...OK
Starting rfd: .OK
Starting HMIPServer: ........OK
Starting ReGaHss: .OK
Starting CloudMatic: disabled
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.71.12.20230826 (raspmatic_ova)
```

**Comment by jens-maus** [comment, marked as answer]
Wird vermutlich mal wieder das bekannte "HmIP-RFUSB nicht sauber genug zusammengelötet" Problem sein. Siehe hier:

https://homematic-forum.de/forum/viewtopic.php?f=31&t=79828

**Comment by silvermd** [comment]
vielen Dank, diesen Thread kannte ich noch nicht.. werde mir die Sachen anschauen und mal einen Kenner über den Stick schauen lassen. evtl. melde  ich mich nochmal;) schönen Abend, vielen Danke

**Comment by silvermd** [comment]
Hallo,

ich möchte kurz eine Rückmeldung geben. Nachdem ich die Lötstellen für gut und sauber bewertet habe (mit dem Handy und Macro Aufnahme konnte man aber ahnen das evtl. der Kontakt nicht gut genug ist), also habe ich nochmals nachgegelötet. 

Jetzt klappt es.. vielen vielen Dank, ich konnte nun die Steckdose (HMIP) und den Türkontakt anlernen. 

Eine Frage am Rand hab ich noch gibt es eine Lösung von einem Funkmodul was ich einfach ans Proxmox per USB anschliessen kann wo ich nicht löten muss, scheint ja nicht gerade meine Stärke zu sein (Übung fehlt halt) Habe auch mal gehört dass der USBStick nicht die beste Leistung hat.

zum Tread aber, vielen Dank mir wurde geholfen und das Problem ist gelöst.
Dennis

**Comment by Baxxy13** [comment]
> gibt es eine Lösung von einem Funkmodul was ich einfach ans Proxmox per USB anschliessen kann wo ich nicht löten muss

Nein, gelötet werden muss immer. 😉 
Aber man muss das ja nicht selber machen sondern kann andere um Hilfe bitten.
Das steht auch so in dem Thread, in Rot und dick.
